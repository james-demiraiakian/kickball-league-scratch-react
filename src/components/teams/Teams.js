import React from 'react';
import PlayerList from '../../view/Player/PlayerList';

export default function Teams({ name, city, state, selected, id }) {
  return (
    <div>
      <h2>{name}</h2>

      {selected && (
        <>
          <h3>
            {city}, {state}
          </h3>
          <h2>Players</h2>
          <PlayerList id={id} />
        </>
      )}
    </div>
  );
}
