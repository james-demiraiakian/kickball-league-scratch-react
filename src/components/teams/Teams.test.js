import { render, screen } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import Teams from './Teams';

it('renders a teams details', async () => {
  const { container } = render(
    <MemoryRouter>
      <Teams name="Namethaniel" city="Citsbourough" state="Stateifornia" selected={true} id={1} />
    </MemoryRouter>
  );
  screen.getByText('Namethaniel');

  expect(container).toMatchSnapshot();
});
