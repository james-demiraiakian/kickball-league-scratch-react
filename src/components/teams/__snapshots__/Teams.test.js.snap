// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a teams details 1`] = `
<div>
  <div>
    <h2>
      Namethaniel
    </h2>
    <h3>
      Citsbourough
      , 
      Stateifornia
    </h3>
    <h2>
      Players
    </h2>
    <h1>
      LOADING
    </h1>
  </div>
</div>
`;
